Makefile

```diff
@@ -68,9 +76,12 @@ distclean: $(patsubst %, %_distclean, ${SUBMODULES}) cw_alarm_agent_distclean
 	rm -rf ${ROOT}/build
 
 include build-infra/cw-deb.mk
-
 .PHONY: deb
 deb: build deb-only
 
+include build-infra/cw-rpm.mk
+.PHONY: rpm
+rpm: build rpm-only
+
 .PHONY: all build test clean distclean
```

Comment:
It turns out that `all` is a target (defined at the top of the file) and the other targets that look like they shouldn't be phony are actually phonies.